docs/guides/cloud.rst

```diff
@@ -49,25 +49,25 @@ create a local project linked to your instance.
     * can contain only Latin alpha-numeric characters or ``-``
     * cannot start with a dash (``-``) or contain double dashes (``--``)
     * maximum instance name length is 61 characters minus the length of your
-      GitHub username (i.e., length of GitHub username + length of instance
+      Org name (i.e., length of Org name + length of instance
```

Comment:
Rebase this on my recent branch ([doc-cloud-connection](https://github.com/edgedb/edgedb/tree/doc-cloud-connection)), and you'll get my fixes on this, which are slightly different from yours.